public/components/marketing-forms/5.html

```diff
@@ -0,0 +1,67 @@
+<!--
+  Heads up! 👋
+
+  Plugins:
+    - @tailwindcss/forms
+-->
+
+<section class="w-full space-y-4 bg-white md:space-y-6">
+  <div>
+    <h2 class="text-2xl font-bold text-gray-900">Profile Settings</h2>
+    <p class="text-sm text-gray-700">Manage your account settings</p>
+  </div>
+  <form action="" class="border-t-2 pt-4 md:pt-6">
+    <div class="space-y-4 divide-y md:space-y-6">
+      <div class="grid gap-4 md:grid-cols-6">
+        <div class="md:col-span-2">
+          <h3 class="text-lg font-medium text-gray-900">Personal Information</h3>
+          <p class="text-sm text-gray-700">Tell us about yourself</p>
+        </div>
+        <div class="space-y-3 md:col-span-4 md:max-w-md">
```

Comment:
Not sure it needs to be `md:max-w-md`, how does it look with just `max-w-md`.

Reply:
I need the `md` selector because it's in that breakpoint where the layout change occurs. Without this class the form would look like this:

![image](https://github.com/markmead/hyperui/assets/46305144/62b55887-e2b3-446e-a7cb-06d4c2ae4531)

Reply:
If it's just `max-w-md` it will apply on all screen sizes and on smaller screens it will be irrelevant. I think try it.

Reply:
Okay will do